DirectProgramming/DPC++FPGA/Tutorials/Features/latency_control/src/latency_control.cpp

```diff
@@ -0,0 +1,123 @@
+//==============================================================
+// Copyright Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+// =============================================================
+#include <CL/sycl.hpp>
+#include <numeric>
+#include <sycl/ext/intel/fpga_extensions.hpp>
+
+// dpc_common.hpp can be found in the dev-utilities include folder.
+// e.g., $ONEAPI_ROOT/dev-utilities/include/dpc_common.hpp
+#include "dpc_common.hpp"
+
+using namespace sycl;
```

Comment:
Please remove the use of namespaces

Reply:
So now `using namespace sycl` is not recommended, and we should type it everywhere in the sample code?

Reply:
Yep